bin/update_dependencies.py

```diff
@@ -21,34 +21,50 @@
 
 if '--no-docker' in sys.argv:
     for python_version in PYTHON_VERSIONS:
-        subprocess.run([
-            f'./env{python_version}/bin/pip-compile',
-            '--allow-unsafe',
-            '--upgrade',
-            'cibuildwheel/resources/constraints.in',
-            '--output-file=cibuildwheel/resources/constraints-python{python_version}.txt',
-        ], check=True)
+        subprocess.run(
+            [
+                f'./env{python_version}/bin/pip-compile',
+                '--allow-unsafe',
+                '--upgrade',
+                'cibuildwheel/resources/constraints.in',
+                '--output-file=cibuildwheel/resources/constraints-python{python_version}.txt',
+            ],
+            check=True,
+        )
 else:
     # latest manylinux2010 image with cpython 2.7 support
     image_runner = 'quay.io/pypa/manylinux2010_x86_64:2021-02-06-3d322a5'
     subprocess.run(['docker', 'pull', image_runner], check=True)
     for python_version in PYTHON_VERSIONS:
         abi_flags = '' if int(python_version) >= 38 else 'm'
         python_path = f'/opt/python/cp{python_version}-cp{python_version}{abi_flags}/bin/'
-        subprocess.run([
-            'docker', 'run', '--rm',
-            '-e', 'CUSTOM_COMPILE_COMMAND',
-            '-v', f'{os.getcwd()}:/volume',
-            '--workdir', '/volume', image_runner,
-            'bash', '-c',
-            f'{python_path}pip install pip-tools &&'
-            f'{python_path}pip-compile --allow-unsafe --upgrade '
+        command = (
+            f'{python_path}pip install pip-tools && '
+            '{python_path}pip-compile --allow-unsafe --upgrade '
             'cibuildwheel/resources/constraints.in '
             f'--output-file cibuildwheel/resources/constraints-python{python_version}.txt'
-        ], check=True)
+        )
+        subprocess.run(
+            [
+                'docker',
+                'run',
+                '--rm',
+                '--env=CUSTOM_COMPILE_COMMAND',
+                "--volume='{os.getcwd()}:/volume'",
```

Comment:
```suggestion
                "--volume={os.getcwd()}:/volume",
```

I don't think the quotes are required - there is no shell to interpret them, and it's not a problem if there are spaces in this path since it'll come into the child process as a single argument anyway. I discovered this in aa12480ff0e5381eca2258a6957aea6af5c46172, note the quotes on the commit message. p.s. already fixed a couple of these in https://github.com/joerick/cibuildwheel/commit/571629607169b4fa5d89304fdda1013ebf310fcd.